homeassistant/components/light/zwave.py

```diff
@@ -141,8 +148,41 @@ def supported_features(self):
         """Flag supported features."""
         return SUPPORT_ZWAVE_DIMMER
 
+    def _set_duration(self, **kwargs):
+        """Set the transition time for the brightness value.
+
+        Zwave Dimming Duration values:
+        0x00      = instant
+        0x01-0x7F = 1 second to 127 seconds
+        0x80-0xFE = 1 minute to 127 minutes
+        0xFF      = factory default
+        """
+        if self.values.dimming_duration is None:
+            if ATTR_TRANSITION in kwargs:
+                _LOGGER.debug("Dimming not supported by %s.", self.entity_id)
+            return
+
+        if ATTR_TRANSITION not in kwargs:
+            self.values.dimming_duration.data = 0xFF
```

Comment:
Yeah, I'll take a look, but at least from a theoretical level it shouldn't. The dimming duration is actually just a parameter that gets sent with the multilevel switch set command.